portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/webcontentdisplay/WebContentDisplay.testcase

```diff
@@ -1095,6 +1095,76 @@ Welcome to Liferay Community Edition Portal 7.4.0 CE GA1''';
 			userEmailAddress = "test@liferay.com");
 	}
 
+	@description = "This is a test for LPS-129944. Out Of Memory Error when web content display widget's configuration is opened."
```

Comment:
Should be configure WCD when selected web content based on template with embedded widget